bench-tps/src/bench.rs

```diff
@@ -711,6 +727,22 @@ mod tests {
         let keypairs = generate_keypairs(&config.id, config.tx_count);
         fund_keys(&clients[0], &config.id, &keypairs, 20);
 
+        let exit = Arc::new(AtomicBool::new(false));
+        let exit1 = exit.clone();
+        // A thread to produce ticks so that the bank's blockhash keeps moving
+        let t_ticker = Builder::new()
```

Comment:
Maybe this should be part of BankClient.

Reply:
hm yeah I think that would be cleaner. I could just remove it altogether. It'll just do 1 batch of bench-tps and then exit. Which is no different from before since it would pretend to do more batches but all the rest would fail with duplicate sigs.